libs/design/src/molecules/feature/feature.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'daff-feature',
+  templateUrl: './feature.component.html',
+  styleUrls: ['./feature.component.scss'],
+  host: {'class': 'daff-feature'},
+  encapsulation: ViewEncapsulation.None,
```

Comment:
changeDetection: ChangeDetectionStrategy.OnPush

Reply:
Resolved in 688add3

Reply:
Resolved in 41f8ddf